metaflow/plugins/datastores/s3_storage.py

```diff
@@ -144,3 +150,34 @@ def iter_results():
                         yield r.key, None, None
 
         return CloseAfterUse(iter_results(), closer=s3)
+
+    def stream_bytes(self, paths, chunk_size=2**1024):
```

Comment:
default chunk_size choice is arbitrary here, mainly for testing purposes. For the UI Backend case this should probably be made configurable.